docs/Index/index.md

```diff
@@ -78,10 +78,29 @@ carch
 
 **How to Install and Use 📦**
 
-<img src="https://raw.githubusercontent.com/smritics/linuxcli/refs/heads/main/docs/assets/arrowdown.webp" width="20" />
```

Comment:
Why did you replace the raw WebP with a PNG? Just to clarify, documentation primarily supports raw images, especially when using Markdown for documentation.

Reply:
Sorry, my mistake. Fixed with the correct PNG.